test/SubdomainWithDCLControllerV2.spec.js

```diff
@@ -196,7 +196,7 @@ describe('DCL Names V2 with DCLControllerV2', function () {
   }
 
   beforeEach(async function () {
-    await loadFixture(deployTokenFixture)
+    await loadFixture(fixture)
```

Comment:
Irrelevant to the PR, but changed the name of the function because it was not correct.